zowe-api.json

```diff
@@ -29,14 +29,23 @@
             "caching-service/build/libs/caching-service.jar": {
                 "target": "components/api-mediation/caching-service.jar",
                 "binary": true
+            },
+            "../api-layer-deploy/libzowe-attls.so": {
```

Comment:
`api-layer-deploy` or rather `common-java`?

Reply:
it doesn't matter much I think, it's still some file that needs to be provided externally, as it is already documented for the zosmf definition and keystore